Makefile

```diff
@@ -107,19 +107,19 @@ test: generate-manifests generate envtest ## Run tests.
 
 .PHONY: e2e
 e2e: 
-	go test ./test/e2e/... -coverprofile cover.out -v
+	go test -tags="e2e" -mod=mod ./test/e2e/... -coverprofile cover.out -test.v
```

Comment:
would `-mod=mod` update the go.mod file when newer dep packages are found?

Reply:
fixed in commit https://github.com/cloudoperators/greenhouse/pull/613/commits/9303ed75f6f5f8209c8e8829cf2d6588ac1c4d1f